### ISSUE TITLE
knife-joyent prints opaque error when client clock is out of sync

### ISSUE THREAD

**Issue by bixu** [joyent/knife-joyent#78]
Error we see from knife-joyent:
```
Exception: NoMethodError: undefined method `body' for nil:NilClass
```

Error we see from smartdc tools:
```
sdc-listnetworks: error (InvalidHeader): Authorization header invalid: clock skew of 319.562s was greater than 300s
```

**Comment by kevinykchan** [comment]
Fixed by foig/fog#3434

**Comment by kevinykchan** [comment]
After fix, we see the following:
```
ERROR: *InvalidHeader*
ERROR: Authorization header invalid: clock skew of 14378.876s was greater than 300s
```
